references/tags.tsv

```diff
@@ -9,7 +9,9 @@ Angermueller2016_dl_review	doi:10.15252/msb.20156651
 Angermueller2016_single_methyl	doi:10.1186/s13059-017-1189-z
 Artemov2016_clinical	doi:10.1101/095653
 Arvaniti2016_rare_subsets	doi:10.1101/046508
+Bach2015_on doi:10.1371/journal.pone.0130140
```

Comment:
The `tags.tsv` parser is fairly picky.  Some of the build errors are being caused by having a space or spaces instead of tabs.  If your text editor coverts tabs to spaces upon saving (like one of mine does), then these errors can arise,

If this type of change is hard to make with your RSI, let me know and I'll edit the tags file myself.  I should have permission to edit your pull request.